chef/cookbooks/monasca/recipes/role_monasca_log_api.rb

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
I'm not sure if monasca-log-api should have a separate role in terms of crowbar roles... Maybe we should put everything into monasca-server role? See https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/crowbar-openstack/blob/monasca/chef/cookbooks/monasca/recipes/role_monasca_server.rb#L19 for reference - monasca-api is included in the monasca-server role (it's commented just for now)